### PR TITLE
Remove tutorials

### PR DESCRIPTION
**Pull request by frankharkins**
This PR removes tutorial notebooks and related tools as they're being moved to a closed-source repository.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -426,14 +426,6 @@ This content lives on https://docs.quantum.ibm.com.
 
 See the section "Syncing content with open source repo" in the internal docs repo's README.
 
-### Deploy tutorials
```
Comment (Eric-Arellano) [inline README.md:429]: You can also remove the h2 header above since there is now only one type of deployment

**Review by Eric-Arellano — APPROVED** [review]
Thanks! There is a new tutorial that we merged this morning. Can you please help to deploy it, then remove it in this PR?

**Comment by frankharkins** [comment]
Draft again as waiting for #1540.

**Comment by frankharkins** [comment]
This is now ready to go

**Review by Eric-Arellano — APPROVED** [review]

**Pull request merged**